test/allowlist_for_publicAPI.json

```diff
@@ -586,6 +586,7 @@
     "threshold_"
   ],
   "torch.nn.init": [
+    "Generator",
```

Comment:
No new entry should be added to this json file.
You can remove this by either adding the appropriate `__all__` entry in the nn.init file or by using `torch.Generator` there to avoid having to make Generator an exposed variable